test/regression/smart_answers_regression_test.rb

```diff
@@ -66,6 +66,10 @@ def teardown_hooks_installed?
         next if self.class.setup_has_run? && !self.class.teardown_hooks_installed?
         stub_content_api_default_artefact
         WebMock.stub_request(:get, WorkingDays::BANK_HOLIDAYS_URL).to_return(body: File.open(fixture_file('bank_holidays.json')))
+        WebMock.stub_request(:get, "#{Plek.new.find('imminence')}/areas/IG6%202BA.json").
+          to_return(body: File.open(fixture_file('imminence/london.json')))
+        WebMock.stub_request(:get, "#{Plek.new.find('imminence')}/areas/B1%201PW.json").
+          to_return(body: File.open(fixture_file('imminence/national.json')))
```

Comment:
@ikennaokpala has addressed this in a new commit titled "Reuse imminence_has_areas_for_postcode in place of web mock" (174d0813933904f23d2e4bfe7852fbc5da36b58d).